bridge/client/app/_components/ktb-evaluation-details/ktb-evaluation-details.component.ts

```diff
@@ -136,6 +136,26 @@ export class KtbEvaluationDetailsComponent implements OnInit, OnDestroy {
       labels: {
         rotation: -45
       },
+      tickPositioner: function() {
+        const positions = [],
+          labelWidth = 70,
+          ext = this.getExtremes(),
+          xMax = Math.round(ext.max),
+          xMin = Math.round(ext.min),
+          maxElements = (document.querySelector('dt-chart')?.clientWidth || labelWidth) / labelWidth,
+          tick = Math.floor(xMax / maxElements) || 1;
+
+        for (let i = xMax; i > xMin; i -= tick) {
+          positions.push(i);
+        }
+        if(positions[positions.length-1] - tick < xMin){ //if the second item is too near to the first one, remove the second one
+          positions[positions.length-1] = xMin;
+        }
```

Comment:
I guess this is responsible for the visual gap on the left side?
![image](https://user-images.githubusercontent.com/56065213/121881743-f2676d00-cd0f-11eb-9249-6ab9daa602b0.png)

I'm not sure if that's a good thing. Why is this?

Reply:
There is not enough space to display the first item, that's why the second one is removed. It could look like this
![image](https://user-images.githubusercontent.com/11599148/121883505-1c219380-cd12-11eb-9147-70011d4b360c.png)

In order to remove this gap, we need to remove the label of the first evaluation. Should we do that?

Reply:
Visually this looks like there is an error with the calculation or spacing. We should obviously show the first and last label, and evenly spread out the labels in between. I'll take a look at the calculation.

Reply:
Okay, I see what's going on here.
This is because we have 50 columns. Therefore we could have 25 labels shown.
If we show the first label though, we will need to hide the second one.

From a visual standpoint, I'd propose to make sure the distance is the same between labels, and therefore, as suggested by @Kirdock, remove the label of the first evaluation. @johannes-b is that okay with you?